app/assets/stylesheets/components/_show_banner.scss

```diff
@@ -0,0 +1,61 @@
+.show-banner {
+  height:90vh;
+  width: 100vw;
+  display: flex;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  flex-direction: column;
+  justify-content: center;
+  align-content: center;
+  gap: 3vh;
+  background:
+    size(cover),
+    linear-gradient(
+      rgba(0, 0, 0, 0.7),
+      rgba(0, 0, 0, 0.7)
+    );
+  color: $white;
+  vertical-align: middle;
+  button {
+    text-decoration: none;
+    color:white;
+    margin: 0 auto;
+  }
+  form {
+    width: 60%;
+    margin: 0 auto;
+  }
+  h1 {
+    text-align: center;
+    margin: 0;
+    text-shadow: $text-shadow;
+    font-size: 2.5rem;
+  }
+}
+// tablets
+@media screen and (min-width: 500px) {
+  .show-banner {
+    h1 {
+      font-size: 3rem;
+    }
+  }
+}
+// average desktop onward
+@media screen and (min-width: 1200px) {
+  .show-banner {
+    height:90vh;
+    h1 {
+      font-size: 3rem;
+    }
+  }
+}
+// bigger monitors
+@media screen and (min-width: 1600px) {
+  .show-banner {
+    height:60vh;
+    h1 {
+      font-size: 3.5rem;
+    }
+  }
+}
```

Comment:
Show banner given different qualities depending on size, so text changes size and remains readable.